### ISSUE TITLE
Wrong name of ubuntu cimage

### ISSUE THREAD

**Issue by titans2016** [Azure/azure-quickstart-templates#2906]
Hi.
Please update template https://github.com/Azure/azure-quickstart-templates/tree/master/docker-neo4j
The name of linux image is wrong.
Template doesn't work.
![2016-12-21 17_49_55-dashboard](https://cloud.githubusercontent.com/assets/13413682/21395681/10cd059e-c7a6-11e6-9d65-6168062b28b0.png)

**Comment by MCKLMT** [comment]
I can confirm the issue. Using 16.10 instead of 15.10 solves the problem.
I don't understand why because using PowerShell, the sku is well valid.

`
PublisherName Offer        Skus              Location
------------- -----        ----              --------
Canonical     UbuntuServer 12.04.2-LTS       eastus
Canonical     UbuntuServer 12.04.3-LTS       eastus
Canonical     UbuntuServer 12.04.4-LTS       eastus
Canonical     UbuntuServer 12.04.5-DAILY-LTS eastus
Canonical     UbuntuServer 12.04.5-LTS       eastus
Canonical     UbuntuServer 12.10             eastus
Canonical     UbuntuServer 14.04-beta        eastus
Canonical     UbuntuServer 14.04.0-LTS       eastus
Canonical     UbuntuServer 14.04.1-LTS       eastus
Canonical     UbuntuServer 14.04.2-LTS       eastus
Canonical     UbuntuServer 14.04.3-LTS       eastus
Canonical     UbuntuServer 14.04.4-LTS       eastus
Canonical     UbuntuServer 14.04.5-DAILY-LTS eastus
Canonical     UbuntuServer 14.04.5-LTS       eastus
Canonical     UbuntuServer 14.10             eastus
Canonical     UbuntuServer 14.10-beta        eastus
Canonical     UbuntuServer 14.10-DAILY       eastus
Canonical     UbuntuServer 15.04             eastus
Canonical     UbuntuServer 15.04-beta        eastus
Canonical     UbuntuServer 15.04-DAILY       eastus
Canonical     UbuntuServer 15.10             eastus
Canonical     UbuntuServer 15.10-alpha       eastus
Canonical     UbuntuServer 15.10-beta        eastus
Canonical     UbuntuServer 15.10-DAILY       eastus
Canonical     UbuntuServer 16.04-alpha       eastus
Canonical     UbuntuServer 16.04-beta        eastus
Canonical     UbuntuServer 16.04-DAILY-LTS   eastus
Canonical     UbuntuServer 16.04-LTS         eastus
Canonical     UbuntuServer 16.04.0-LTS       eastus
Canonical     UbuntuServer 16.10             eastus
Canonical     UbuntuServer 16.10-DAILY       eastus
Canonical     UbuntuServer 17.04-DAILY       eastus
`

Maybe @gatneil can help?

**Comment by gatneil** [comment]
Yeah, the Ubuntu Server 15.10 images have been removed from the platform; I do have a work item in my backlog to remove them from the repo, but Im trying respond to open PRs first :). As for why it doesn't work even though the SKU shows up in Powershell: if you query the versions for that SKU, you'll notice that there are none! (Azure CLI sample below) I guess the moral of the story is: always check that there exists at least one version for the SKU you wan to use ;).

```
negat@jumpbox:~/$ azure vm image list westus Canonical UbuntuServer 15.10
info:    Executing command vm image list
+ Getting virtual machine images (Publisher:"Canonical" Offer:"UbuntuServer" Sku: "15.10" Location:"westus")
warn:    No virtual machine images found
info:    vm image list command OK
```

**Comment by MCKLMT** [comment]
@gatneil By what version would you like to replace the 15.10?

**Comment by MCKLMT** [comment]
@gatneil The following command gives us all the available images for Ubuntu. 

`Get-AzureRmVMImageSku -Location WestEurope -PublisherName "Canonical" -Offer "UbuntuServer" | Get-AzureRmVMImage`

I will make a pull request.

**Comment by gatneil** [comment]
Looks like that PR replaces both 14.* and 15.* with 16.04-LTS; ideally we would only replace 15.* with 16.04-LTS, as 14.* still has an LTS version in the platform :)

**Comment by gatneil** [comment]
Actually, I think my comment was wrong; it looks like it's changing 16.04.*-LTS to 16.10, 15.04 to 16.04-LTS, and 14.04.*-LTS to 14.04.5-LTS; Ideally we'd leave the 14.* and 16.* options untouched, simply removing the 15.* options

**Comment by MCKLMT** [comment]
@gatneil Fixed!
